sale_stock_medical_prescription/models/medical_prescription_order_line.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class MedicalPrescriptionOrderLine(models.Model):
+    _inherit = 'medical.prescription.order.line'
+
+    dispense_uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='Dispense UoM',
+        help='Dispense Unit of Measure',
+        required=True,
+        related='medicament_id.uom_id',
```

Comment:
**Labelling this as a notable change**
* Previously, the dispense_uom_id had a _default=_ attribute that pointed to a [_default_dispense_uom_id](https://github.com/OCA/vertical-medical/pull/89/commits/a7422679edcf3cd3b94f12b84e28f9f8be0ac970#diff-d52542e328b24456f88777e961babd72L68) method
* This was causing problems however, because if the default context uom was of a different category as the medicament's (e.g. grams => units, cannot convert), this would raise an error.
* This related field fixed that
* A possible idea might be to add a try-except block to the method instead of removing it, not sure.